src/server/game/Entities/Player/Player.h

```diff
@@ -1625,17 +1623,17 @@ class Player : public Unit, public GridObject<Player>
     [[nodiscard]] Unit* GetSelectedUnit() const;
     [[nodiscard]] Player* GetSelectedPlayer() const;
 
-    void SetTarget(uint64 /*guid*/) override { } /// Used for serverside target changes, does not apply to players
-    void SetSelection(uint64 guid);
+    void SetTarget(ObjectGuid /*guid*/ = ObjectGuid::Empty) override { } /// Used for serverside target changes, does not apply to players
```

Comment:
Why add `= ObjectGuid::Empty`?

Reply:
It's an overriden function - default is empty.

Reply:
Okay